pages/18f/how-18f-works/research-guidelines.md

```diff
@@ -106,6 +106,10 @@ participants, especially from underserved communities, and helps build trust.
 Without providing incentives and compensation, we are likely to hear only from
 the people who can afford to volunteer their time.
 
+## Planning for compensation
+
+There are a few things to consider and steps to take to ensure that you or your team are able to provide compensation when it is most appropriate. Taking time up front for planning and approvals will help avoid delays and frustrations for teams and research participants alike.
+
 ### Who is not eligible for compensation?
```

Comment:
Per discussion in co-working - this is likely the least important part but it comes up front

Perhaps this section becomes footnote content and we link to it for folks that need it.